src/connectors/pagination/__tests__/connectPagination-test.js

```diff
@@ -115,7 +115,11 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/pagination/
     }
 
     widget.render({
-      results: new SearchResults(helper.state, [{}]),
+      results: new SearchResults(helper.state, [
+        {
+          nbPages: 250,
```

Comment:
There was an issue swallowed by `lodash/range` implementation with this test. `nbPages` was `undefined` so the Paginator `end` value when `undefined` and the `end` value computed and given to `lodash/range` was `NaN`. Adding this value to the search results makes the test more consistent with real-life cases.